client/blocks/importer/wordpress/import-everything/pre-migration/style.scss

```diff
@@ -0,0 +1,118 @@
+@import "~calypso/blocks/import/style/base";
+@import "../style.scss";
+
+.import__pre-migration.import__import-everything--redesign {
+	max-width: 100%;
+}
+
+.import__pre-migration {
+
+	padding: 0 20px;
```

Comment:
Did you try using `rem` scss mixin for transforming px to rem?

This line could be like: `padding: 0 rem(20px)`.

For padding, margin, font-size, line-height, etc., we should rely on the rem unit.